terraform/README.md

```
@@ -0,0 +1,140 @@
+
+GCPでOwner権限があれば、ほとんどのリソースの作成・削除は問題なく行える。Editor権限だとIAMリソースの変更は不可、それ以外は大体変更できる。
+
+```bash
+gcloud auth application-default login
+```
+
+### 手順
+
+Terraform CLIをインストール
+
+* https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)